pkg/controller/node/nodecontroller_test.go

```diff
@@ -597,6 +597,157 @@ func TestMonitorNodeStatusUpdateStatus(t *testing.T) {
 	}
 }
 
+func TestMonitorNodeStatusUpdatePodStatus(t *testing.T) {
```

Comment:
Because the unit test is triggering monitorNodeStatus function, and this function triggers the pod mark down.